docs/abi-spec.rst

```diff
@@ -81,6 +81,24 @@ It is possible to form tuples of tuples, arrays of tuples and so on.  It is also
 .. note::
     Solidity supports all the types presented above with the same names with the exception of tuples. The ABI tuple type is utilised for encoding Solidity ``structs``.
```

Comment:
Should have squashed this note into the table.

Reply:
Created #5420